### ISSUE TITLE
Introductions and initial discussion

### ISSUE THREAD

**Issue by rdocking** [rdocking/fusebench#2]
Hi everyone! As we get ready for the event this weekend, I'd like to start preparing potential issues that team members can take on. To help with that (and to start getting to know each other), could people please comment on this issue with a brief overview of:

- General background info
- Main languages you're comfortable working with
- Whether you'll be attending in Vancouver or working remotely (and if remote, which time zone?)
- Whether you're comfortable working with Git/GitHub/Slack

Also - I've recently become aware of some new projects that substantially overlap with the original idea for this project (in particular, the recently-published [chimeraviz](https://github.com/stianlagstad/chimeraviz) seems quite similar). I still think there's room for an interesting project in this space, but we may need to re-scope and re-focus a bit. 

As far as organization and documentation, I was hoping to mainly use this Git repo and the [Slack channel](https://hackseq.slack.com/messages/C7HEWA2NT). 

Thanks so much for participating in HackSeq!

cc: @eacton @devanjan31 @hiraksarkar @wilcas @SweetiePi @shibuantony @stefpiatek

**Comment by wilcas** [comment]
Hello All,

My background is in both computer science and biology, I'm a first year MS/PhD student currently working with eQTL studies. I'm hoping to branch out into different ways to infer environmental effects on gene expression/detect and catalog said interactions using different ML approaches (I plan to have more of a scope after doing a few smaller projects in my lab). 

My main experience is with python and bash, I am also currently working in MATLAB (okay with Octave and Julia) and I've done some smaller projects/homework assignments in C/C++.  My most recent project was an automated/parallel benchmark framework for somatic variant callers. It was written using a python API/query framework on the DNAnexus cloud platform (which sits on AWS and more cloud providers), and it's main features were independent/parallel execution of jobs and the ability to weave in new variant callers to test without touching the source code. Before that I was doing some work replacing some code for a Max Parsimony Phylogeny algorithm. This involved implementing (not designing) a multi-commodity flow framework for building a max parsimony tree using a python binding for SCIP, and then implementing the same script in C++ to update an existing version of the project written using the CPLEX ILP solving library.  

I'll be attending in Vancouver, but I have an exam on Friday at 4:00 PM about a block from the Life Sciences Institute, so I will be out for that hour. I am comfortable working with git/GitHub and I used it pretty extensively last summer. I have used slack in the past but I don't have as much of an intuition about its workflow.

Looking forward to hacking with you all, and thanks @rdocking for leading

P.S. BRIEFLY worked with fusioncatcher during my last internship (try and see if you can find my tiny bug fixes)

**Comment by alexsweeten** [comment]
Hey guys!

My name's Alex and I'm a first year MS student in bioinformatics. Currently I'm working in SFU's Computational Epidemiology lab, where I am researching ML clustering methods to analyze the pathogenicity of certain E. coli strains. 

I'm comfortable with Python, R, and C++, as well as using Git and Slack. I'll be in Vancouver Friday and Saturday but will possibly be unavailable Sunday. See you all soon!

**Comment by stefpiatek** [comment]
Hey everyone,

I'm Stef and I'm a trainee clinical bioinformatician (purely diagnostic 
within a national genetics laboratory) where we do germline variant 
analyses. I did a PhD in molecular biology where I got into 
bioinformatics using R to do some microarray gene expression analyses.

At the moment probably use python the most but comfortable in R and 
bash, along with git/github and slack. I'll be working from sunny 
England so it should be interesting to see how that works! At an all day 
meeting on Friday and won't get back time for the first day but over the 
weekend have blocked out the time to work on it.

**Comment by hiraksarkar** [comment]
Hi everyone, 

I am a computer science Ph.D student from Stony Brook University, specializing in computational biology. I am fluent in C++, Python. Not great with R, still learning. 

I will be collaborating remotely. 
I am comfortable with slack/github/gitter. 

PS: Call me Hirak

**Comment by eacton** [comment]
Hello! 

I'm Erica and I am a bioinformatician in the Hancock lab at UBC, working in microbial diseases and immunity research. I analyze transcriptome data - from alignment to network and pathway analysis -  with an emphasis on visualizing large, complex datasets. My background is in Genome Science and Technology.

I mainly work in R and use command line for genomics tools and small tasks. 
I am comfortable with GitHub and Slack.
I will be attending in person. Woot!

Looking forward to working with you all!

**Comment by rdocking** [comment]
Great - thanks all for your comments! To reciprocate: I'm a PhD candidate in the Experimental Program (which is in the Department of Medicine at UBC). Since 2008, I've been in and around the Genome Sciences Centre at the BC Cancer Agency - first as a computational biologist working on genome assembly, then helping to start a clinical diagnostics group at the [CCG](http://ccgenomics.com/). I left that group to pursue my PhD with Dr. Aly Karsan, where I'm interested in using trnascriptome data to better understand and diagnose myeloid malignancies (like acute myeloid leukemia). 

My day-to-day work these days is mostly done in R, but I used to be mostly a Python person and am not so terribly rusty with it.

**Comment by rdocking** [comment]
Quick summary to date:

| Name  | GitHub handle | Languages                        | Location        | Timezone  | Availability |
|-------|---------------|----------------------------------|-----------------|-----------|--------------|
| Rod   | @rdocking     | R, Python, etc                   | Vancouver       | UTC-08:00 | Fri-Sun      |
| Will  | @wilcas       | Python, bash, MATLAB, C/C++, etc | Vancouver       | UTC-08:00 | Fri-Sun      |
| Alex  | @SweetiePi    | Python, R, C++                   | Vancouver       | UTC-08:00 | Fri-Sat      |
| Stef  | @stefpiatek   | Python, R, bash                  | London          | UTC+00:00 | Sat-Sun      |
| Hirak | @hiraksarkar  | C++, Python, etc                 | Stony Brook, NY | UTC-05:00 | Fri-Sun      |
| Erica | @eacton       | R, command line                  | Vancouver       | UTC-08:00 | Fri-Sun      |
| Devanjan     | @devanjan31   | Python             | India                | UTC+05:30   | Fri-Sun             |
| Shibu     | @shibuantony  | TBD                                 | India                | UTC+05:30    | Fri-Sun    |
| Jishnu | | | | | |
| Joyce | | | | | |

**Comment by rdocking** [comment]
Because we have several remote participants, and it sounds like everyone is roughly familiar with Git and Slack, I'll propose that we keep most of our work in these two channels. 

I'm going to start revising/refining some of the scope for the project based on how many people we have (and some of the recent papers that have been published). I'll do this in a branch and submit a PR for people to review/discuss (so we can see if that workflow will work for us). 

In terms of computational resources, we should have login credentials soon for the shared computing platform we'll be using soon. 

Any questions/comments?

**Comment by hiraksarkar** [comment]
I am not very familiar with the exact timeline and the format of the contest. Are going to receive a set of goals other than the agenda we have?

**Comment by rdocking** [comment]
Hi Hirak - as far as timeline, the 'official' schedule is at: http://www.hackseq.com/schedule . Those are roughly the times that I'll be on-site (and on-line) for the event. Whatever time you can make is greatly appreciated.

I'm not sure I'd describe the event as a 'contest' - we'll be presenting and discussing with the other groups, but I'm not sure if there are any prizes or anything like that. 

As far as specific goals, I'm going to be spending some time this afternoon and tomorrow starting to carve the project up into some more actionable chunks, based on the team's makeup. I'll try to submit a first PR this afternoon with some ideas.

**Comment by hiraksarkar** [comment]
Sounds great !! looking forward to it.

**Comment by devanjan31** [comment]
Hi,
Sorry for the late reply.
I'm Devanjan Chakravarty from VIT University, India and I am pursuing my
undergraduate in the field of biotechnology.
I've good experience working in several projects/assignments using basic
bioinfirmatics tools based on python. I've also used IBM Watson in a
hackathon to develop a diagnostic tool for mouth cancer.
I'll be participating from India.
Hope we have a good time.

Best wishes and regards
Devanjan Chakravarty

**Comment by devanjan31** [comment]
Hi,
Sorry for the late reply.
I'm Devanjan Chakravarty from VIT University, India and I am pursuing my
undergraduate in the field of biotechnology.
I've good experience working in several projects/assignments using basic
bioinfirmatics tools based on python. I've also used IBM Watson in a
hackathon to develop a diagnostic tool for mouth cancer.
I'll be participating from India.
Hope we have a good time.

Best wishes and regards
Devanjan Chakravarty

On 20 Oct 2017 12:37 am, "Devanjan Chakravarty" <devanjan31@gmail.com>
wrote:

Hi,
Sorry for the late reply.
I'm Devanjan Chakravarty from VIT University, India and I am pursuing my
undergraduate in the field of biotechnology.
I've good experience working in several projects/assignments using basic
bioinfirmatics tools based on python. I've also used IBM Watson in a
hackathon to develop a diagnostic tool for mouth cancer.
I'll be participating from India.
Hope we have a good time.

Best wishes and regards
Devanjan Chakravarty
